pkg/apis/flowcontrol/v1alpha1/doc.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:conversion-gen=k8s.io/kubernetes/pkg/apis/flowcontrol
+// +k8s:conversion-gen-external-types=k8s.io/api/flowcontrol/v1alpha1
+// +k8s:defaulter-gen=TypeMeta
+// +k8s:defaulter-gen-input=../../../../vendor/k8s.io/api/flowcontrol/v1alpha1
+
+// +groupName=flowcontrol.apiserver.k8s.io
+
+package v1alpha1 // import "k8s.io/kubernetes/pkg/apis/flowcontrol/v1alpha1"
```

Comment:
no package comment here

Reply:
This doc file still lacks a package comment.   It should have a package comment.